vtex-apps/product-summary
# Fix product summary css, hover on mobile and default schema props

#### What is the purpose of this pull request?

1. Fix the Product Summary CSS (details on screenshots)
2. Fix #15 (Hide list price if equals selling price)
3. Fix #18 (Hover doesn't exist on mobile and tablets)
4. Fix #23 (Default Schema props)

#### What problem is this solving?

Fix Summary issues and it's styling

#### How should this be manually tested?
[Access workspace](https://estacio--storecomponents.myvtex.com/) and see the styling and the Schema props. Access from a smartphone or tablet to see the buy button if it's not hidden completely.

#### Screenshots or example usage
![image](https://user-images.githubusercontent.com/15948386/39206286-62d4d70a-47d3-11e8-999a-73b5d1bc7512.png)

Summary specification: 
![image](https://user-images.githubusercontent.com/15948386/39206315-700db0cc-47d3-11e8-944c-95dd969f90a5.png)


#### Types of changes
